.github/workflows/build.yaml

```diff
@@ -67,6 +68,7 @@ jobs:
         run: nix-build -A outputs.checks.x86_64-linux.${{ steps.params.outputs.attr }}-mysql
 
       - name: Build Xdebug extension
+        if: ${{ steps.params.outputs.major != 8 && steps.params.outputs.minor != 3 }}
```

Comment:
This will disable it on all PHP 8 versions and PHP 7.3.

You want `!(major = 8 && minor = 3)`